src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -4548,14 +4634,14 @@ private void VisitMemberAccess(BoundExpression node, BoundExpression receiverOpt
             return null;
         }
 
-        private int GetNullableOfTValueSlot(TypeSymbol containingType, int containingSlot)
+        private int GetNullableOfTValueSlot(TypeSymbol containingType, int containingSlot, out Symbol fieldOrProperty)
```

Comment:
>Symbol fieldOrProperty [](start = 95, length = 22)

Isn't this always a MethodSymbol?

Reply:
It's the associated property. Renamed to `valueProperty`.

---
In reply to: [250796376](https://github.com/dotnet/roslyn/pull/32729#discussion_r250796376) [](ancestors = 250796376)